rasa/cli/utils.py

```diff
@@ -124,6 +129,68 @@ def create_output_path(
         return os.path.join(output_path, file_name)
 
 
+def button_to_string(button, idx=0):
```

Comment:
```suggestion
def button_to_string(button: Dict[Text, Any], idx: int = 0) -> Text:
```

this might not be the exact way to type this 😅 but let's add typing